### PR TITLE
Personal website revisions

### PR DESCRIPTION
**Pull request by MaxWofford**
Based on input from https://gist.github.com/remixz/3bf575b32bb96be43ea0

### CODE REVIEW

File: workshops/personal_website/README.md [hunk 1]
```diff
@@ -214,33 +214,30 @@ Let's see what our website looks like so far:
...
 > ![](img/html_add_name.gif)
 
-Notice that that the HTML code you write is immediately updated in the preview.
-That's why it's called a **"Live Preview"**
+Notice that that the HTML we write is immediately updated in the preview. That's
+why it's called a **"Live Preview"**
 
 ### 6) Adding a Description
 
-Underneath on a new line, write one short sentence about yourself. I love my job
```
Comment (remixz) [inline workshops/personal_website/README.md]: Why was this line removed? We should keep a text description of every step, since if we only have it in the GIF, it'll be inaccessible to people who can't view images, or are using a screen reader.

File: workshops/personal_website/README.md [hunk 2]
```diff
@@ -491,34 +488,33 @@ we can share with anyone on the internet! We'll the previously mentioned GitHub
...
   - `git commit -m "Initial commit"`
   - `git push`
-2. GitHub will now ask you for your username and password.
-  - Go ahead and enter your username and then press the enter.
-  - Then enter your password and press enter. _The characters won't show up on
-    the screen, but rest assured, you are still typing._
-3. Now try to view your website by going to
-   `USERNAME.github.io/personal_website` (make sure to replace `USERNAME` with
-   your actual GitHub username)
+2. GitHub will now ask for our username and password.
```
Comment (remixz) [inline workshops/personal_website/README.md]: This is one instance where I think "your" might be better, since GitHub credentials aren't shared with anyone else.

**Comment by remixz** [comment]
:+1: This looks great! I left a few notes on the file. Once those are resolved, this should be good to merge.

File: workshops/personal_website/README.md (hunk 1 above)
Comment (MaxWofford) [inline workshops/personal_website/README.md]: Thanks for catching that! Will fix that right now

File: workshops/personal_website/README.md (hunk 2 above)
Comment (MaxWofford) [inline workshops/personal_website/README.md]: :+1:

**Comment by MaxWofford** [comment]
@remixz Thanks for the comments! Just pushed the new changes.

**Comment by remixz** [comment]
:tada:
